packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -25,35 +33,51 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 
-### Empty Mapping
+### Empty mapping
 
 <Description of={emptyMappingStories} />
 <Controls of={emptyMappingStories.emptyMapping} />
 
 ## Usage
 
-### Best Practices
+### Best practices
+
+-   Provide a mapping for every expected record value. Because grouping is
+    performed on the record value, non-mapped record values can result in
+    multiple groups without group labels.
+    -   To improve grouping behavior of a value that shouldn't be rendered in
+        table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
+        for that value's group row.
+-   Avoid having multiple values that map to the same text because grouping and
+    sorting is done on the record value rather than the mapped text.
+
+#### Choosing a mapping style
 
--   Provide a mapping for every expected record value. Because grouping is performed on the record value, non-mapped record values can result in multiple groups without group labels.
-    -   To improve grouping behavior of a value that shouldn't be rendered in table cells, use a <Tag name={mappingEmptyTag}/> element to provide text for that value's group row.
--   Avoid having multiple values that map to the same text because grouping and sorting is done on the record value rather than the mapped text.
+| Best practice                                                                                                                                                                                  | Example                                      |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| **Icon and text, with a text header** <br/> Default configuration where icons exist for the mapping set                                                                                        | <img src={iconTextTextHeader} />             |
+| **Icon-only, with an icon header** <br/> Use only when table width is at a premium and all the icons in the set have a clear and unambiguous meaning                                           | <img src={iconOnlyIconHeader} />             |
+| **Icon-only, with a text header** <br/> Use when header text provides enough context for all the icons in the set to have a clear and unambiguous meaning                                      | <img src={iconOnlyTextHeader} />             |
+| **Text-only, with a text header** <br/> Use when there isn't an icon for each mapping value                                                                                                    | <img src={textOnlyTextHeader} />             |
+| **Icon-only column beside a text-only column** <br/> Use to be compatible with existing columns, or when the end-user is likely to remove the text-only column in a custom table configuration | <img src={iconOnlyPlusTextOnlyTextHeader} /> |
```

Comment:
How about a note about mixing mapping types? Could we explicitly recommend NOT mixing text mappings with icon mappings in the same column since it would result in misalignment? 

And what about mixing `text-hidden` values on mappings within the same column? We do it in the example above where the spinner mapping sets `text-hidden` but the icon mappings don't. But I don't think we'd recommend a mixture of icon mappings where some show text and some don't.

(I'm borderline on whether this is over explaining; I'm open to letting our clients use common sense and not documenting this at all)

Reply:
Added explicit guidance not to mix and match except for spinners.